### PR TITLE
balloons: support for L2 cache cluster allocation

### PR DESCRIPTION
**Pull request by askervin**

### CODE REVIEW

**Comment by askervin** [comment]
This PR depends on #383

**Comment by askervin** [comment]
(Documentation added as the last commit.)

**Review by kad — APPROVED** [review]
LGTM

**Review by fmuyassarov — APPROVED** [review]
LGTM :+1:

File: test/e2e/policies.test-suite/balloons/n4c16/test13-cacheclusters/balloons-2cpu-cacheclusters.cfg [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+  {"cpusets": ["0-1", "2-3", "4-5", "6-7"], "level": 2, "size": "4M", "kind": "u"},
+  {"cpusets": ["8-9", "10-11", "12-13", "14-15"], "level": 2, "size": "4M", "kind": "u"},
+
+  {"cpusets": ["0-1", "2-3", "4-5", "6-7"], "level": 1, "size": "256k", "kind": "d"},
+  {"cpusets": ["0-1", "2-3", "4-5", "6-7"], "level": 1, "size": "128k", "kind": "i"},
+
+  {"cpusets": ["8-9", "10-11", "12-13", "14-15"], "level": 1, "size": "256k", "kind": "d"},
+  {"cpusets": ["8-9", "10-11", "12-13", "14-15"], "level": 1, "size": "128k", "kind": "i"}
+  ]'''
```
Comment (klihub) [inline test/e2e/policies.test-suite/balloons/n4c16/test13-cacheclusters/balloons-2cpu-cacheclusters.cfg:39]: Is this cache configuration override interesting/useful ? Doesn't it provide the same grouping as the socket and the physical cores ?

**Review by klihub — APPROVED** [review]

**Pull request merged**